app/move/fields/additional-information.js

```diff
@@ -1,11 +1,16 @@
 const additionalInformation = {
+  id: 'additional_information',
```

Comment:
Could you import the common version of this file into this field?

Reply:
Probably - I was leaning towards making an issue and doing it later, but will look properly now

Reply:
Rejigged to use the actual `additional_information` field as the base for the others